### PR TITLE
Adding contrast to csss accessibility snippets

### PR DESCRIPTION
**Pull request by simin4950**
# What Changed

Added CSS less-contrast snippet

# Why

Added for https://github.com/intuit/accessibility-snippets/issues/37
Allowing developers to create applications with the flexibility to change contrast for their users. 

Todo:
- [ ] Add Semantic Version Label 
- [x] Add tests/Tested
- [x] Add docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 ### CSS snippet
 
 - reduced motion - adds a reduced motion snippet with prefix `transition`
+- less-contrast - adds less contrast snippet with prefix `less-contrast`
```
Comment (kendallgassner) [inline README.md:17]: Can you add the safari note here too:


```suggestion
🚨 Safari Only Queries  🚨 
- less-contrast - adds less contrast media snippet with prefix `less-contrast`
```

Comment (simin4950) [reply README.md]: done :)

File: snippets/css-snippets.code-snippets [hunk 2]
```diff
@@ -11,5 +11,19 @@
...
         ],
         "description": "Adds reduced motion media query anytime transition is added to css file"
+    },
+
+    "contrast": {
+        "prefix": [
+            "less-contrast",
+        ],
+        "body": [
+            "less-contrast: \t$1",
```
Comment (kendallgassner) [inline snippets/css-snippets.code-snippets:21]:
```suggestion
```

Comment (simin4950) [reply snippets/css-snippets.code-snippets]: done :)

**Review by kendallgassner — APPROVED** [review]
⭐

**Pull request merged**